src/net/client/xfr.rs

```diff
@@ -0,0 +1,1147 @@
+//! Parsing of AXFR/IXFR response messages for higher level processing.
```

Comment:
This doesn't seems to be a client transport so it should go somewhere else. Maybe net::xfr could be a place for XFR functionality.

Reply:
Moved to `net::xfr`.